docs/content/en/docs/soarca-extensions/fin-python-library.md

```diff
@@ -0,0 +1,184 @@
+---
+title:  Fin Python Library
+description: >
+    Documentation of the Python Fin library
+categories: [extensions, architecture]
+tags: [fin, python]
+weight: 2
+date: 2024-04-10
+---
+
+For documentation about the fin protocol we refer the reader to de documention page of [SOARCA](https://cossas.github.io/SOARCA/docs/soarca-extensions/fin-protocol/)
+
+## Quick Start
+To use the SOARCA Fin library use the following command to install the library using pip:
+```bash
+pip install soarca-fin-library
+```
+
+### Example
+An example on how to use the library is given below. 
+For more examples and the source code, we will refer to the Github page of the [project](https://github.com/COSSAS/SOARCA-FIN-python-library).
+```python
+def capability_pong_callback(command: Command) -> ResultStructure:
```

Comment:
Maybe define the variables like finId, capabilityId and use them at the correct places?